patches/chrome-browser-chrome_content_browser_client.cc.patch

```diff
@@ -1,8 +1,19 @@
 diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
-index 196205f32d149db75ea3edeb01b3521735cafa08..39897ad441746b4259c6bf02effa81911206824e 100644
+index 196205f32d149db75ea3edeb01b3521735cafa08..38f93d1df536a478c32306e415f40900ff4c7759 100644
 --- a/chrome/browser/chrome_content_browser_client.cc
 +++ b/chrome/browser/chrome_content_browser_client.cc
-@@ -782,6 +782,16 @@ void GetGuestViewDefaultContentSettingRules(
+@@ -461,6 +461,10 @@
+ #include "chrome/services/file_util/public/mojom/constants.mojom.h"
+ #endif
+ 
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OS_LINUX)
```

Comment:
@simonhong can you remove these patches in a follow-up? The chromium_src subclass override I mentioned on slack would work well here

Reply:
in fact in this case you subclass the ViewsDelegateLinux and you won't even need BraveBrowserMainExtraPartsViewLinux

Reply:
Done in https://github.com/brave/brave-core/pull/230